DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Reports/_FilterDisplayCard.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Reports
+@model ActivityFilterModel
+
+@{
+  var dateFormat = "dd/MM/yyyy";
+}
+
+<div class="nhsuk-card">
+  <div class="nhsuk-card__content nhsuk-u-padding-bottom-0">
+    <div class="nhsuk-card__description">
+      <dl class="nhsuk-summary-list details-list-with-button">
+        <div class="nhsuk-summary-list__row details-list-with-button__row">
+          <dt class="nhsuk-summary-list__key">
+            Job group
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.JobGroupName" />
+        </div>
+
+        @if (!string.IsNullOrEmpty(Model.CourseCategoryName)) {
+          <div class="nhsuk-summary-list__row details-list-with-button__row">
+            <dt class="nhsuk-summary-list__key">
+              Course category
+            </dt>
+            <partial name="_SummaryFieldValue" model="@Model.CourseCategoryName" />
+          </div>
+        }
+
+        @if (!string.IsNullOrEmpty(Model.CustomisationName)) {
+          <div class="nhsuk-summary-list__row details-list-with-button__row">
+            <dt class="nhsuk-summary-list__key">
+              Course
+            </dt>
+            <partial name="_SummaryFieldValue" model="@Model.CustomisationName" />
+          </div>
+        }
+
+        <div class="nhsuk-summary-list__row details-list-with-button__row">
+          <dt class="nhsuk-summary-list__key">
+            Date range
+          </dt>
+          <dd class="nhsuk-summary-list__value">
+            @Model.StartDate.ToString(dateFormat) - @Model.EndDate.ToString(dateFormat)

```

Comment:
Could we set the DateRange field on the model so it can be unit tested? (can just be a derived field)
And the date format can just be taken from the date helper.